upload-cloud-storage/README.md

```diff
@@ -0,0 +1,194 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# upload-cloud-storage
+
+This action uploads files/folders to a [GCS][gcs] bucket. This is useful when
```

Comment:
Use "Google Cloud Storage (GCS)" the first time in case people don't know what GCS is.

Reply:
done